src/styles/vars.scss

```diff
@@ -0,0 +1,7 @@
+$primary-bg-color: #f3f3f3;
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-main-color: #00acdc;
+$background-card-color: #fff;
+$hover-color: #34568b;
+$column-count: 1;
```

Comment:
This variable will not be necessary anymore after the fixes from previous comments, you can delete it :)